ghlicense/cmd.py

```diff
@@ -396,7 +396,7 @@ def main():
             update_license("https://joinup.ec.europa.eu/sites/default/files/inline-files/EUPL%20v1_2%20EN(1).txt", chosen_license,
                            '(https://img.shields.io/badge/License-EUPL%20v1.1-blue.svg)](https://joinup.ec.europa.eu/page/eupl-guidelines-faq-infographics)')
         else:
-            print('License {license} not found!'.format(license=chosen_license))
+            print('No license provided'.format(license=chosen_license))
```

Comment:
This is wrong, the license maybe can be provided and not exist at the same time so this not fix the issue

Reply:
Yes, I've got it.

I've fix it